plugins/integration/services/IntegrationService.php

```diff
@@ -50,7 +50,13 @@ public function notifyAction($id)
 		$batchJob = BatchJobPeer::retrieveByPK($id);
 		$invalidJobId = false;
 		$invalidKs = false;
-		if(!$batchJob)
+		
+		if(!self::validateKs($batchJob))
```

Comment:
@shpiNi  - first need to check if(!$batchJob) and then if(!self::validateKs($batchJob))